src/viur_cli/package.py

```diff
@@ -141,6 +142,13 @@ def scriptor(version, target, profile):
 
     real_version, download_url = get_version_info("scriptor", version)
 
+    old_version = conf.get("builds.scriptor.version")
+
+    if old_version == real_version.strip("v"):
+        if not click.confirm(f"You have already installed the latest version of scriptor {old_version}. "
```

Comment:
If you install an specific version like: `viur package install scriptor default v1.0.0` which is not the latest this info would be wrong.

```suggestion
        if not click.confirm(f"You have already installed the version {old_version} of scriptor. "
```